tests/integration/test_config.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+
+import pytest as pytest
+from pytest_operator.plugin import OpsTest
+
+from .helpers import (
+    CHARM_SERIES,
+    DATABASE_APP_NAME,
+    get_leader_unit,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.group(1)
+@pytest.mark.abort_on_fail
+async def test_config_parameters(ops_test: OpsTest) -> None:
+    """Build and deploy one unit of PostgreSQL and then test config with wrong parameters."""
+    # Build and deploy the PostgreSQL charm.
+    async with ops_test.fast_forward():
+        charm = await ops_test.build_charm(".")
+        await ops_test.model.deploy(
+            charm,
+            num_units=1,
+            series=CHARM_SERIES,
+            config={"profile": "testing"},
+        )
```

Comment:
The k8s charm has slightly different parameters (trust, resource). Better to use the [build_and_deploy](https://github.com/canonical/postgresql-k8s-operator/blob/main/tests/integration/helpers.py#L57) helper